src/App.js

```diff
@@ -1,5 +1,7 @@
 import { useMutation } from '@tanstack/react-query';
 import { useRef, useState } from 'react';
+import Banner from './Banner';
+import { OP_SEPOLIA_CHAIN_ID } from './index';
```

Comment:
loop detected. index included App, App includes index.
For now you can hardcode chain id as number where it is used and not do any constants.

Reply:
removed the constant and hard-coded value